Formula/c/cloud-provider-kind.rb

```diff
@@ -0,0 +1,20 @@
+class CloudProviderKind < Formula
+  desc "Cloud provider for KIND clusters"
+  homepage "https://github.com/kubernetes-sigs/cloud-provider-kind"
+  url "https://github.com/kubernetes-sigs/cloud-provider-kind/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "ba3fad794370b1ff14e608edfa5883cb455e1e6e89d6a63187c5f47bf4e23251"
+  license "Apache-2.0"
+  head "https://github.com/kubernetes-sigs/cloud-provider-kind.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    # Should error out as creating a kind cluster requires root
+    status_output = shell_output("#{bin}/cloud-provider-kind -h 2>&1")
+    assert_match "Usage: cloud-provider-kind", status_output
```

Comment:
Can it test something more than help output?
The comment above is probably misleading.

Reply:
Oh, because this file is a copy of a kind.rb, I forgot to change this. 😅
Originally, I wanted to test the `cloud-provider-kind` command because macOS requires sudo, but Linux doesn't require sudo to run it.